source/comply/custom-terms-of-service.rst

```diff
@@ -46,12 +51,3 @@ Why isn't this feature in Team Edition for GDPR compliance?
 Terms of service is presented to users on login and account creation, and available to users at all times in the link available by going to **System Console > Site Configuration > Customization**.
```

Comment:
```suggestion
Terms of Service is presented to users on login and account creation. It's available to users at all times in the link available by going to **System Console > Site Configuration > Customization**.
```

Reply:
This isn't very clear - is the link available to users using that path, or is that the link that they'll see that's stored in that location?